apps/docs/src/guide/provider/index.md

```diff
@@ -0,0 +1,47 @@
+# Provider
+
+The [`Provider`](../../api/Account/Provider.md) class lets you connect to a Fuel node ([_*local*_](../introduction/getting-started.md#connecting-to-a-local-node) or [_*external*_](../introduction/getting-started.md#connecting-to-the-testnet)) and interact with it, encapsulating common client operations in the SDK.
```

Comment:
I think this could be made a bit more general about the kind of functions you can perform with a provider. Then info on connecting using it.